scripts/create-aws-resources.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+import json
+from os import environ
+
+from django.conf import settings
+import boto3
+import botocore
+
+AWS_ACCESS_KEY_ID = environ.get('AWS_ACCESS_KEY_ID')
+AWS_SECRET_ACCESS_KEY = environ.get('AWS_SECRET_ACCESS_KEY')
+# TODO REGION needs to get into an environment variable
+REGION = 'us-east-1'
```

Comment:
Actually, this is already in injected into the env. https://github.com/lewfish/raster-foundry/blob/b7113316e3c7c14ba8d9046420061d76f7143665/deployment/ansible/group_vars/all#L4